src/cc_helpers.cc

```diff
@@ -110,29 +116,6 @@ certifier::framework::cc_trust_data::cc_trust_data(const string& enclave_type, c
   }
 }
 
-certifier::framework::cc_trust_data::cc_trust_data() {
-  cc_basic_data_initialized_ = false;
-  cc_policy_info_initialized_= false;
-  cc_policy_store_initialized_ = false;
-  cc_service_key_initialized_ = false;
-  cc_service_cert_initialized_ = false;
-  cc_service_platform_rule_initialized_ = false;
-  cc_sealing_key_initialized_ = false;
-  cc_provider_provisioned_ = false;
-  x509_policy_cert_ = nullptr;
-  cc_is_certified_ = false;
-
-  for (int i = 0; i < num_certified_domains_; i++) {
-      if (certified_domains_[i] != nullptr) {
-        delete certified_domains_[i];
-        certified_domains_[i] = nullptr;
-      }
-  }
-  delete certified_domains_;
```

Comment:
This and delete on L127 seem like destructor code.

To me, seems like this old code was mixing-up logic / code for constructor and destructor.